src/osgGA/TerrainManipulator.cpp

```diff
@@ -126,7 +126,7 @@ void TerrainManipulator::setByMatrix(const Matrixd& matrix)
 
             _rotation.set(0,0,0,1);
 
-            hitFound = true;
```

Comment:
Another example if (!state) { state = true; ) pattern is used here and not safe to change in case later changes assume that the pattern is maintained with all blocks.  It's not safe to remove, or even comment out.  It's not a performance issue leaving it in.